AbilityChanger/Abilities/WallJump.cs

```diff
@@ -9,14 +9,30 @@ public class WallJump : AbilityManager {
             On.HeroController.DoWallJump += WallJumpAbilityTrigger;
         }
         public void WallJumpAbilityTrigger(On.HeroController.orig_DoWallJump orig, HeroController self){
-            if(isCustom()){
-                this.handleAbilityUse();
+            
+            //Trigger
+            if(currentAbility.hasTrigger()){
+                HandleTrigger();
             } else {
+                //Start
+                HandleStart();
```

Comment:
this is sus code.
only if we don't have trigger then we start our ability ?

imo should be 
```cs
if(currentAbility.isCustom())
{
            if(currentAbility.hasTrigger()){
                HandleTrigger();
            } 
            HandleStart();
} else {
          orig(self);
}
```